site/topics/sorting/sorting.rst

```diff
@@ -135,6 +135,11 @@ Insertion Sort
     * Elements are inserted into their proper relative location to the elements already within the sorted collection
     * Any subsequent insertion cannot disrupt the ordering of the whole list if it is being inserted in its proper location
 
+        * For example, consider the list `[1, 4, 7, 9]`
+        * If we need to insert the number `6`, it obviously goes between the `4` and `7` since ``4 < 6 < 7`` --- `[1, 4, 6, 7, 9]`
+        * The order of the elements ``1`` and ``4`` remain unchanged, and similarly with ``7`` and ``9``
```

Comment:
Felt like I _could_ discuss the example more, but I really hope their intuition takes over.